packages/react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -379,29 +404,8 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
         margin: '1px',
       },
     ],
-    dropdownItemHeader: [
-      globalClassnames.dropdownItemHeader,
-      {
-        ...fonts.medium,
-        fontWeight: FontWeights.semibold,
-        color: semanticColors.menuHeader,
-        background: 'none',
-        backgroundColor: 'transparent',
-        border: 'none',
-        height: DROPDOWN_ITEM_HEIGHT,
-        lineHeight: DROPDOWN_ITEM_HEIGHT,
-        cursor: 'default',
-        padding: '0 8px',
-        userSelect: 'none',
-        textAlign: 'left',
-        selectors: {
-          [HighContrastSelector]: {
-            color: 'GrayText',
-            ...getHighContrastNoAdjustStyle(),
-          },
-        },
-      },
-    ],
+    dropdownItemHeader: [...dropdownHeaderStyle],
```

Comment:
nit: does `dropdownHeaderStyle` need to be spread here? Looks like the same usage as `dropdownItemSelected` on L388 and we're not using spread there.

Reply:
Good point, a spread is not necessary here - just made the change!